src/main/java/com/three_tech_solutions/slot_app/services/implementations/StudentServiceImpl.java

```diff
@@ -99,4 +102,17 @@ public void deleteStudent(UUID studentId){
                 })
                 .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "El estudiante no existe."));
     }
-}
+
+    @Override
+    public StudentResponse updateStudent(UUID studentId, UpdateStudentRequest studentUpdated) {
+        validatePlanDetail(studentUpdated.getPlanType(), studentUpdated.getPaymentDay());
+        return studentRepository.findById(studentId)
+                .map(student -> {
+                    StudentMapper.updateStudent(student, studentUpdated);
```

Comment:
Aca estamos accediendo al metodo mediante su clase estatica, como te decia, como el StudentMapper es un bean, deberiamos acceder a los metodos mediante una relacion que ya tenemos (fijate que en la linea 28 tenemos una dependencia con el StudentMapper) 


```suggestion
                    studentMapper.updateStudent(student, studentUpdated);
```

De esta manera, en vez de acceder mediante el metodo estatico accedemos mediante la referencia